src/views/History.tsx

```diff
@@ -1,10 +1,62 @@
-import React, {useState} from 'react';
-import {View, TextInput, FlatList} from 'react-native';
+import React, {useState, useEffect} from 'react';
+import {
+  View,
+  TextInput,
+  FlatList,
+  Button,
+  TouchableOpacity,
+  Text,
+  StyleSheet,
+} from 'react-native';
 import HistoryMailItem from '../components/HistoryMailItem';
+import DatePicker from 'react-native-date-picker';
+import {REACT_APP_API_BASE_URL} from '@env';
+
+function dateToStringLongFormat(date: Date): string {
+  return date.toLocaleDateString('en-US', {
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+  });
+}
+
+function dateToEpochTime(date: Date, isStartDate: boolean): string {
+  const month = (date.getMonth() + 1).toString().padStart(2, '0');
```

Comment:
why +1?

Reply:
Javascript getMonth() returns the month number from 0 to 11 so +1 is to make sure it's 1 to 12

Reply:
that's kinda dumb